Exec/SCIENCE/ecsn/input_ecsn

```diff
@@ -51,7 +56,7 @@ maestro.use_divu_firstdt = true
 # 0 = Interior   3 = Symmetry
 # 1 = Inflow     4 = Slipwall
 # 2 = Outflow    5 = NoSlipWall
-maestro.lo_bc = 3 3 3
+maestro.lo_bc = 2 2 2
 maestro.hi_bc = 2 2 2
```

Comment:
for an octant, you need to have reflecting boundaries on the lower faces, to enforce the symmetry.

Reply:
I am not using octant now. I am using 512 resolution and tagging.f90. So I set "maestro.lo_bc" back to 2 2 2.

Reply:
oh sorry, I didn't see that that was commented out.